packages/neuron-ui/src/widgets/Pagination/index.tsx

```diff
@@ -42,32 +41,40 @@ const Pagination = ({ count, pageNo, onChange, pageSize }: PaginationProps) => {
   const end = Math.min(count, pageNo * pageSize)
   const range = t(`${I18N_PATH}.range`, { start, end, count })
 
+  const disableToHead = pageNo === 1 || pageCount === 0
+  const disableToEnd = pageNo >= pageCount
+
   return (
     <div role="presentation" className={styles.container} onClick={handlePageNoClick}>
       <div className={styles.range}>{range}</div>
       <div className={styles.navigator} role="navigation" arial-label="pagination">
-        <span
-          role="button"
-          data-page-no={1}
-          tabIndex={pageNo === 1 ? -1 : 0}
-          className={styles.toPrev}
-          data-disabled={pageNo === 1}
-          title={t(`${I18N_PATH}.first-page`)}
-        >
-          <ArrowToEndIcon />
-        </span>
-        <span
-          role="button"
-          data-page-no={pageNo - 1}
-          tabIndex={pageNo === 1 ? -1 : 0}
-          className={styles.toPrev}
-          data-disabled={pageNo === 1}
-          title={t(`${I18N_PATH}.previous-page`)}
-        >
-          <ArrowToNextIcon />
-        </span>
+        <div className={styles.arrowBlock}>
+          <div
```

Comment:
Same question as https://github.com/nervosnetwork/neuron/pull/2537#discussion_r1044252690, why not use `button` directly, it would be good for semantic presentation